src/qt/walletmodel.cpp

```diff
@@ -36,10 +40,20 @@
 #include <QSet>
 #include <QTimer>
 
+#include <boost/foreach.hpp>
+#include <map>
+#include <univalue.h>
 
-WalletModel::WalletModel(const PlatformStyle *platformStyle, CWallet *_wallet, OptionsModel *_optionsModel, QObject *parent) :
-    QObject(parent), wallet(_wallet), optionsModel(_optionsModel), addressTableModel(0),
+/* We need the following functions declared for use in the QT UI */
+extern UniValue name_new(const JSONRPCRequest& request);
+extern UniValue name_update(const JSONRPCRequest& request);
+extern UniValue name_firstupdate(const JSONRPCRequest& request);
+extern UniValue gettransaction(const JSONRPCRequest& request);
```

Comment:
Please declare these in `src/wallet/rpcwallet.h` (not sure about `gettransaction`, maybe some other header is more appropriate for that).